<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JanelaEstatisticas</class>
 <widget class="QWidget" name="JanelaEstatisticas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>913</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="grafico" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>541</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid gray;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbCalorias">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>93</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calorias</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPeso">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>93</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Peso</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
	background-color: rgb(230, 255, 213);
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }</string>
   </property>
   <property name="title">
    <string>Nutrientes</string>
   </property>
   <widget class="QCheckBox" name="cbProteina">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Proteina</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFerro">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ferro</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCarboidrato">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Carboidrato</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbCalcio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbFibraAlimentar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fibra alimentar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbColesterol">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Colesterol</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaA">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina A</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaB1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina B1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaB2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina B2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaB6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina B6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaB12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina B12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaC">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAcidoFolico">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Acido Folico</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbPotassio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Potassio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMagnesio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>93</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Magnesio</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaD">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina D</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaE">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina E</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbVitaminaK">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitamina K</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="cbFormato">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>74</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Dias</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Semanas</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Meses</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Anos</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar por:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>321</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    
	background-color: rgb(230, 255, 213);
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top center; /* position at the top center */
     padding: 0 3px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #FFOECE, stop: 1 #FFFFFF);
 }</string>
   </property>
   <property name="title">
    <string>Filtrar data</string>
   </property>
   <widget class="QDateEdit" name="deDe">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="deAte">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>110</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>De</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>172</x>
      <y>35</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>At√©</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Selecione o que voce deseja que apareca no grafico</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btPlotar">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>92</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Plotar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btPlotar</sender>
   <signal>clicked()</signal>
   <receiver>JanelaEstatisticas</receiver>
   <slot>plotar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>plotar()</slot>
 </slots>
</ui>
